webcompat/static/js/lib/bugform-prefill.js

```diff
@@ -1,52 +0,0 @@
-/* exported prefillForm */
```

Comment:
Do you think you could split out all the old form deletion stuff into a separate commit? 

It's sort of a pain, but it would be nice to not have these 2 unrelated changes coupled together. If you want to crate an issue for that, it could be referenced in https://github.com/webcompat/webcompat.com/projects/12 as well.

Reply:
Yeah, good point, I've moved them to the issue Karl created https://github.com/webcompat/webcompat.com/pull/3394